ovos_dinkum_listener/service.py

```
@@ -194,7 +188,25 @@ def __init__(self, on_ready=on_ready, on_error=on_error,
 
-        self.hotwords = hotwords or HotwordContainer(self.bus)
+        verifiers_cfg  = self.config.get("listener", {}).get("ww_verifiers", {})
+        verifier_plugs = {}
+        for plug_type, plug in find_wake_word_verifier_plugins().items():
+            cfg = verifiers_cfg.get(plug_type, {})
+            if not cfg.get("enabled", True): # plugins are enabled by default if installed, unless disabled in config
+                LOG.debug(f"wakeword verifier plugin disabled: {plug_type}")
+                continue
+            try:
+                verifier_plugs[plug_type] =  plug(config=cfg)
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)